code/modules/clothing/suits/armor.dm

```diff
@@ -182,7 +182,7 @@
 	icon_state = "acolyte"
 	item_flags = THICKMATERIAL | COVER_PREVENT_MANIPULATION
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
-	armor = list(melee = 30, bullet = 30, energy = 25, bomb = 25, bio = 100, rad = 80)
+	armor = list(melee = 30, bullet = 30, energy = 25, bomb = 20, bio = 100, rad = 80)
```

Comment:
No

Reply:
Eh, fair, I legit have no idea what's the general use and application of this and five bomb isn't much difference anyways